packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents/plan-language-tags-guesser.ts

```diff
@@ -0,0 +1,156 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { v2 as CloudTranslate } from '@google-cloud/translate';
+import { Stripe } from 'stripe';
+
+/**
+ * This lib attempts to detect the (human) language used in a Stripe plan's
+ * metadata.
+ *
+ * The language will be determined with one or more of the following:
+ *  - the language detected by Google Cloud Translation from the plan's product
+ *    details metadata
+ *  - the plan's title
+ *  - the plan's currency
+ *
+ * There is no direct connection between a currency and a localised language,
+ * but with the special case of Switzerland (the only one at the time of
+ * writing), this is the sure way to detect for Swiss language tags.
+ */
+
+const MIN_CONFIDENCE = 0.51;
+export const PLAN_EN_LANG_ERROR = 'Plan specific en metadata';
+const { Translate } = CloudTranslate;
+const translate = new Translate();
+let locales: string[];
+
+const initLocales = (x: string[]) => {
+  locales = x;
+};
```

Comment:
It's the minimal (least risky) change to support the supported languages, which would be the default list anyway, as your converter script is already importing it indirectly through the auth server configs.  Being to able to set the list also make testing easier.  And for a script that we hopefully will run only a few times, I didn't feel the need to update the tests because the supported languages is not a superset of the list used in the tests.

I do admit that it's not the most optimal code!